docs/framework/data/adonet/ef/index.md

```diff
@@ -4,6 +4,6 @@ ms.date: "03/30/2017"
 ms.assetid: a437041f-6899-4ae7-96ce-aabf528d7205
 ---
 # ADO.NET Entity Framework
-The [msdn.com/data/ef](http://msdn.microsoft.com/data/ef) site is now the main location for the Entity Framework content.  
+The [docs.microsoft.com/en-us/ef/index](https://docs.microsoft.com/en-us/ef/) site is now the main location for the Entity Framework content.  
```

Comment:
The links should not include "en-US" -- with it, the user won't get localized content depending on the browser's culture.
The text link can also just be docs.microsoft.com/ef/. The index isn't necessary.